win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1354,7 +1355,7 @@ CURSORICON_LoadFromFileW(
     /* Check for .ani. */
     if (memcmp( bits, "RIFF", 4 ) == 0)
     {
-        UNIMPLEMENTED;
+        hCurIcon = (HANDLE)CURSORICON_GetCursorDataFromANI(&cursorData, bits, filesize, fuLoad);
```

Comment:
Why `(HANDLE)` cast needed?

Reply:
I think it's indeed not needed to cast it, since it doesn't work properly. I guess I should just check the `CURSORICON_GetCursorDataFromANI` function with all required parameters and then do something with `hCurIcon` variable, something like
`if(!CURSORICON_GetCursorDataFromANI(&cursorData, bits, filesize, fuLoad)`
`{`
`    /* do something if the function returns FALSE */`
`}`
`else`
`{`
`    /* do something with hCurIcon if it returns TRUE */`
`}`
But I wonder what it should be exactly. This turned out to be more difficult than I thought at first. Sorry. :sweat_smile: 
I did the cast earlier only because it has been done in Wine by this way. I think that the converting `CURSORICON_GetCursorDataFromANI` into `HANDLE` type is also wrong.